app/controllers/issues_controller.rb

```diff
@@ -1,49 +1,45 @@
 class IssuesController < ApplicationController
+  before_action :authenticate_user!
+
   def index
-      #@books = Book.all
-      #check_book_availability
-      @books = Book.order(:id).page params[:page]
+    @books = Book.order(:id).page params[:page]
   end
 
-  def new 
-      @issue = Issue.new
+  def new
+    @issue = Issue.new
   end
 
   def create
     @issue = Issue.new(book_id: params[:book_id], user_id: params[:user_id])
     if @issue.save
-      flash.now[:notice] = "Issue book successfully"
-      redirect_to issues_path, notice: "Issue book succesfully"
+      flash.now[:notice] = 'Issue book successfully'
+      redirect_to issues_path, notice: 'Issue book succesfully'
     else
       render :new, status: :unprocessable_entity
     end
   end
 
-  # def buy 
+  # def buy
   #   #byebug
   #   user = User.new(buy_params)
   #   if user.save
   #     redirect_to issues_path
-  #   else 
+  #   else
   #     render :buy, status: :unprocessable_entity
   #   end
 
   # end
 
-
-
   private
 
   def book_params
     params.require(:book).permit(:name, :author)
```

Comment:
why you define books params in issue controller

Reply:
changes done